src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,56 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        String resultRead = readFrom(fromFileName);
+
+        String[] lines = resultRead.split(System.lineSeparator());
+        for (String line : lines) {
+            String[] arraySplit = line.split(",");
+            if (arraySplit[0].equals("supply")) {
+                supply += Integer.parseInt(arraySplit[1]);
+            } else {
+                buy += Integer.parseInt(arraySplit[1]);
+            }
+            StringBuilder resultBuilder = new StringBuilder();
+            resultBuilder.append("supply").append(",").append(supply).append(System.lineSeparator())
+                    .append("buy").append(",").append(buy).append(System.lineSeparator())
+                    .append("result").append(",").append(supply - buy)
+                    .append(System.lineSeparator());
+            String reportStatistic = resultBuilder.toString();
+            writeToFile(toFileName, reportStatistic);
```

Comment:
lets move this logic to separate method too(calculateStatistic for example). We will read file and get String[] lines, then pass them as input parameter to calculateStatistic which will return String with report, which we will pas to writeToFile method